grpc/tests/src/test/get_balance.rs

```diff
@@ -141,8 +142,8 @@ fn get_balance_should_work() {
         *DEFAULT_ACCOUNT_ADDR,
         runtime_args! {
             TRANSFER_ARG_TARGET => *BOB_ADDR,
-            TRANSFER_ARG_AMOUNT => *TRANSFER_AMOUNT_1
-
+            TRANSFER_ARG_AMOUNT => *TRANSFER_AMOUNT_1,
+            TRANSFER_ARG_ID => <Option<u64>>::None
```

Comment:
Suggestion: Something like

```rust
type TransferID = u64;
```

or even `type TransferID = Option<u64>`

could be helpful across codebase where there are arguments constructed in various tests, contracts, traits, etc

Reply:
I'm not sure I see the value of adding a type alias here. It still has to be treated/constructed as an `Option<u64>`, so this would just be a notational shorthand.

Reply:
We can follow up on this offline.  Not totally opposed, but I would like to follow up as a separate PR if we decide to do it.